var/spack/repos/builtin/packages/h5bench/package.py

```diff
@@ -58,3 +59,33 @@ def cmake_args(self):
         ]
 
         return args
+
+    @run_after("install")
+    def setup_build_tests(self):
+        launcher = self.mpi_launcher()
```

Comment:
I don't see this variable used anywhere in this method.  Are you intending to use it elsewhere or am I missing something?

Reply:
> I don't see this variable used anywhere in this method. Are you intending to use it or am I missing something?

Nvm.  After testing I now understand the launcher problem with the `test_h5bench`.  You need another `filter_file` here to replace the launcher (e.g., `mpirun`) in the `"command"` with the full path to the launcher.